stubs/pyinstaller/PyInstaller/depend/analysis.pyi

```diff
@@ -0,0 +1,19 @@
+# https://pyinstaller.org/en/stable/hooks.html#the-pre-safe-import-module-psim-api-method
+
+# The documentation explicitely mentions that "Normally you do not need to know about the module-graph."
+# However, some PyiModuleGraph typed class attributes are still documented as existing in imphookapi.
+from _typeshed import Incomplete
+from typing_extensions import LiteralString
+
+class PyiModuleGraph:  # incomplete
```

Comment:
Given the reason I haven't completed this stub (see comment at the top of the file). Does the `# incomplete` comment apply here?
I have the same question for similar reasons in `build_main.pyi`.